apps/gp2-frontend/src/users/export.ts

```diff
@@ -0,0 +1,90 @@
+import { caseInsensitive, CSVValue } from '@asap-hub/frontend-utils';
+import { gp2 } from '@asap-hub/model';
+import { formatDate } from '@asap-hub/react-components';
+/* eslint-disable-next-line import/no-unresolved */
+import { Stringifier } from 'csv-stringify/browser/esm';
+
+export const userFields = {
+  firstName: 'First name',
+  lastName: 'Last name',
+  email: 'Email',
+  region: 'Region',
+  location: 'Location',
+  role: 'GP2 role',
+  degrees: 'Degrees',
+  onboarded: 'Onboarded',
+  primaryPosition: 'Primary position',
+  secondaryPosition: 'Secondary position',
+  terciaryPosition: 'Terciary position',
+  projects: 'Projects',
+  workingGroups: 'Working groups',
+  fundingStreams: 'Funding streams',
+  contributingCohorts: 'Contributing cohorts',
+  createdDate: 'Ativated account',
+};
+
+type UserCSV = Record<keyof typeof userFields, CSVValue>;
+
+export const MAX_SQUIDEX_RESULTS = 200;
+
+export const userToCSV = (output: gp2.UserResponse): UserCSV => ({
+  firstName: output.firstName,
+  lastName: output.lastName,
+  email: output.email,
+  region: output.region,
+  location: `${output.country}${output.city ? ', ' : ''}${output.city || ''}`,
+  role: output.role,
+  degrees: output.degrees?.sort(caseInsensitive).join(',\n'),
+  onboarded: output.onboarded ? 'Yes' : 'No',
+  primaryPosition: output.positions[0]
+    ? `${output.positions[0].role} in ${output.positions[0].department} at ${output.positions[0].institution}`
+    : undefined,
+  secondaryPosition: output.positions[1]
+    ? `${output.positions[1].role} in ${output.positions[1].department} at ${output.positions[1].institution}`
+    : undefined,
+  terciaryPosition: output.positions[2]
+    ? `${output.positions[2].role} in ${output.positions[2].department} at ${output.positions[2].institution}`
+    : undefined,
```

Comment:
Could be a function that takes a positions and an index, then returns the formatted string or not.
terciary should be tertiary